Automattic/wp-calypso
# Themes Showcase: Remove upgrade to premium banner

Related to:
- https://github.com/Automattic/dotcom-forge/issues/4537

## Proposed Changes

* Removed the banner as requested

## Testing Instructions



* Open  the calypso live link
* Navigate to the LiTS with on a site with a Free or Personal plan
* You should not see the upgrade banner

Before:
![Captura de pantalla 2023-11-23 a las 11 18 30](https://github.com/Automattic/wp-calypso/assets/1989914/3384be89-eb2a-490a-90b0-4156e6e628ed)

After:
![Captura de pantalla 2023-11-23 a las 11 23 57](https://github.com/Automattic/wp-calypso/assets/1989914/38d165f6-f841-4c9b-b3fb-072ec2b9a9a3)


## Pre-merge Checklist
